### PR TITLE
Create custom resource quotas for okd-team

### PR DESCRIPTION
**Pull request by sherine-k**

### CODE REVIEW

File: cluster-scope/base/core/namespaces/okd-team/okd-quotas.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: large
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/okd-team/okd-quotas.yaml:4]: Please rename the resource to `okd-custom` (filename should be `resourcequota.yaml`)

**Review by tumido — CHANGES_REQUESTED** [review]
Only a nit, otherwise lgtm

File: cluster-scope/base/core/namespaces/okd-team/okd-quotas.yaml (hunk 1 above)
Comment (sherine-k) [reply cluster-scope/base/core/namespaces/okd-team/okd-quotas.yaml]: Sure!
thanks so much

File: cluster-scope/base/core/namespaces/okd-team/resourcequota.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: large
```
Comment (tumido) [inline cluster-scope/base/core/namespaces/okd-team/resourcequota.yaml:4]: Sorry, I probably was not clear enough... You're half way there :smile: 

Please rename the resource to `okd-custom`
```suggestion
  name: okd-custom
```

**Review by tumido — CHANGES_REQUESTED** [review]

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**